app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\User;
+
+class UserController extends Controller
+{
+	const ROLES = 'enum.user_roles';
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+		$users = User::where('role', config(self::ROLES)['sys_admin'])->get();
```

Comment:
Creo que en este User Story estamos considerando poder hacer CRU de todos los tipos de usuario, no solamente admins del sistema. @melytc could probably confirm this.